### PR TITLE
Fix removing listeners when an event was never added

### PR DESCRIPTION
**Pull request by juandopazo**
Buenas!

So... We're using this @ Yahoo. Can't say where. 😛 

Let me know if you need anything to merge this.

### CODE REVIEW

**Comment by juandopazo** [comment]
Ur build is failing btw

File: test/test.js [hunk 1]
```diff
@@ -84,6 +84,12 @@ describe('Jvent', function () {
 			expect(listener).not.toHaveBeenCalled();
 		});
 
+        it('Does not throw when trying to remove a listener that was never added', function() {
+            expect(function() {
+                emitter.off('something', listener);
+            }).not.to.throw();
```
Comment (pazguille) [inline test/test.js]: The syntax should be `.not.toThrow()`. Could you update this line?

Comment (juandopazo) [inline test/test.js]: Right. Jasmine, not chai.

**Comment by juandopazo** [comment]
Updated.

**Pull request merged**